lib/src/proto_op_store.rs

```diff
@@ -56,7 +44,8 @@ pub struct ProtoOpStore {
 }
 
 impl ProtoOpStore {
-    pub fn init(store_path: PathBuf) -> Self {
+    #[allow(dead_code)]
+    fn init(store_path: PathBuf) -> Self {
```

Comment:
Why retain this at all?

Reply:
No good reason :) Deleted.

Reply:
Ah, I had to remove the tests because they used this code, but that seems fine.